src/current/_includes/v23.1/ui/sessions.md

```diff
@@ -73,6 +73,13 @@ The **Cancel session** button ends the session. The client that holds this sessi
 
   A list of the most recent transaction fingerprint IDs, represented in hexadecimal, executed by this session. The fingerprint ID is a link to the [Transaction Details page]({{ link_prefix }}ui-transactions-page.html#transaction-details-page) for the transaction. When you click the link, in order to fetch and render the transaction details, the global date range is changed to the session's start and end time.
 
+### Closed session cache
+
+Details about closed sessions are stored in memory in a closed session cache. This cache is controlled by the [`sql.closed_session_cache.capacity`] ({{ link_prefix }}cluster-settings.html#setting-sql-closed-session-cache-capacity) [cluster setting]({{ link_prefix }}cluster-settings.html) which is the maximum number of sessions in the cache and defaults to `1000`. The data in the cache is deleted according to the [`sql.closed_session_cache.time_to_live`]({{ link_prefix }}cluster-settings.html#setting-sql-closed-session-cache-time-to-live) cluster setting which is the maximum time to live, in seconds, and defaults to  `3600`.
```

Comment:
the data is delete when they reach the TTL OR the limit is reached (on this second case it deletes the older data to be able to include new ones).
Current sentence makes it sounds that we only add up to 1000 and deletes only according to the TTL, but is a combination of both

Reply:
modified with suggested clarification